vars/seleniumExecuteTests.groovy

```diff
@@ -102,11 +101,9 @@ void call(Map parameters = [:], Closure body) {
 
         // Inject config via env vars so that scripts running inside selenium can respond to that
         config.dockerEnvVars = config.dockerEnvVars ?: [:]
-        config.dockerEnvVars.PIPER_CONTAINER_PORT_MAPPING = new JsonOutput().toJson(config.containerPortMappings)
-        config.dockerEnvVars.PIPER_DOCKER_NAME = new JsonOutput().toJson(config.dockerName)
-        config.dockerEnvVars.PIPER_DOCKER_IMAGE = new JsonOutput().toJson(config.dockerImage)
-        config.dockerEnvVars.PIPER_SIDECAR_NAME = new JsonOutput().toJson(config.sidecarName)
-        config.dockerEnvVars.PIPER_SIDECAR_IMAGE = new JsonOutput().toJson(config.sidecarImage)
+        config.dockerEnvVars.PIPER_SELENIUM_HOSTNAME = config.dockerName
+        config.dockerEnvVars.PIPER_SELENIUM_WEBDRIVER_HOSTNAME = config.sidecarName
```

Comment:
How is the behaviour with SeleniumHub? In that case are these variable still relevant?

Reply:
In that case, one would probably use the `karma-selenium-grid-launcher` or specific Karma plugins of the respective Cloud service, which either allow to hard-code the URL or have the URL built-in. Anyway, the variables wouldn't be of any use then.

Reply:
For a JavaScript adapter, we should probably check if the SeleniumHub credentials are set and if yes, assume that Karma should connect to that rather than the headless browser in the Jenkins Library.